types/idle-js/index.d.ts

```diff
@@ -0,0 +1,89 @@
+// Type definitions for idle-js 1.2
+// Project: https://github.com/soixantecircuits/idle-js
+// Definitions by: Jeff Peirson <https://github.com/jpeirson>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare class IdleJs {
+    /**
+     * Creates a new unstarted idle tracking instance.
+     *
+     * @param options Initial options
+     * @throws
+     */
+    constructor(options?: IdleJs.Options);
+
+    /**
+     * Starts tracking idle state.
+     *
+     * @returns this instance for call chaining
+     */
+    start(): IdleJs;
```

Comment:
change `IdleJs` to `this`
```suggestion
    start(): this;
```